river/tree/splitter/exhaustive_splitter.py

```diff
@@ -194,3 +170,30 @@ def _search_for_best_split_option(
         )
 
         return current_best_option
+
+
+class Node:
```

Comment:
Same as before, what about `ExhaustiveNode`? 

This is optional.

Reply:
Renamed in commit [3b4fed1](https://github.com/online-ml/river/pull/488/commits/3b4fed13fee51b7f77a3b5226ac24f27894f44ff).